README.md

```diff
@@ -47,6 +47,9 @@ and hence own the full stack what they develop.
   tool to ensure a common coding style across teams.
     - **article: [PEP 8][pep8] (time investment: 2 hours)**
     - **article: [PEP 20 aka The Zen of Python][pep20] (time investment: 2 min)**
+- Learn about Python virtual environments (virtualenv) in order to
+  keep your local development environment sane.
+    - **tutorial: [Python Guide: Virtual Environments.][virtualenv] (time investment: 10 min)**
```

Comment:
I also really like this [article](http://www.dabapps.com/blog/introduction-to-pip-and-virtualenv-python/) (time investment: 15 min)